packages/services/src/userFeedback/userFeedback.ts

```diff
@@ -8,26 +8,30 @@ import { UserFeedback, UserFeedbackCreateInput } from '@fc/types'
 export const createUserFeedback = async (
   userFeedback: UserFeedbackCreateInput,
   token: string,
+  recaptchaToken?: string,
 ) => {
   if (!userFeedback) {
     throw new Error('feedback field is required')
   }
 
   await Mutation.post<UserFeedback, UserFeedbackCreateInput>(
     'user-feedbacks',
-    userFeedback,
+    {
+      ...userFeedback,
+      recaptchaToken,
+    } as UserFeedbackCreateInput,
```

Comment:
_You don't have to change anything, it's just a note._

New TS operator `satisfies` is more strict and recommended.

```suggestion
    } satisfies UserFeedbackCreateInput,
```

Reply:
i remembered. but I specifically used 'as'. because when recaptchaToken was included, it is no longer satisfied the UserFeedbackCreateInput object. I made it so ts would ignore it.